websocket/output/conf.yaml

```
@@ -0,0 +1,6 @@
+redis:
+  url: redis://127.0.0.1:6379
+  ttl: 3600
+gcs:
+  bucket_name: yrs-dev
+  endpoint: http://localhost:4443
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)